content/docs/instrumenting/exporters.md

```diff
@@ -143,18 +144,26 @@ possible.
 
 ## Directly instrumented software
 
-Some third-party software already exposes Prometheus metrics natively, so no
-separate exporters are needed:
+Some third-party software directly uses a Prometheus client library and exposes
```

Comment:
Perhaps use a wording like the following to emphasize that this is more about instrumentation than exposition:
"Some third-party software is directly instrumented using a Prometheus client library and exposes..."